dhis-2/dhis-services/dhis-service-dxf2/src/test/java/org/hisp/dhis/dxf2/events/security/EnrollmentSecurityTest.java

```diff
@@ -233,27 +238,16 @@ public void testUserWithDataReadWriteNoOrgUnit()
 
         injectSecurityContext( user );
 
-        assertEquals( ImportStatus.ERROR, enrollmentService.addEnrollment(
-            createEnrollment( programA.getUid(), maleA.getUid() ), ImportOptions.getDefaultImportOptions() )
-            .getStatus() );
-
-        assertEquals( ImportStatus.ERROR, enrollmentService.addEnrollment(
-            createEnrollment( programA.getUid(), maleB.getUid() ), ImportOptions.getDefaultImportOptions() )
-            .getStatus() );
-
-        assertEquals( ImportStatus.ERROR, enrollmentService.addEnrollment(
-            createEnrollment( programA.getUid(), femaleA.getUid() ), ImportOptions.getDefaultImportOptions() )
-            .getStatus() );
-
-        assertEquals( ImportStatus.ERROR, enrollmentService.addEnrollment(
-            createEnrollment( programA.getUid(), femaleB.getUid() ), ImportOptions.getDefaultImportOptions() )
-            .getStatus() );
```

Comment:
Where did these go?

Reply:
They were never called. I removed the `expected = SomeException.class` and ran these tests to figure out which of the calls actually threw the exception.